### PR TITLE
Cleaning a little bit the example plugin

### PR DESCRIPTION
**Pull request by xRoier**
- Deleted the private constructor.
- Moved down ``base.OnEnabled()`` and ``base.OnDisabled()``.

### CODE REVIEW

File: Exiled.Example/Example.cs [hunk 1]
```diff
@@ -41,21 +37,18 @@ private Example()
         /// <inheritdoc/>
         public override void OnEnabled()
         {
-            base.OnEnabled();
-
             RegisterEvents();
-
```
Comment (iRebbok) [inline Exiled.Example/Example.cs:47]: Revert that like breaker. It looks better, you don't get confused in logs and method calls with a large number of logs.

File: Exiled.Example/Example.cs [hunk 2]
```diff
@@ -41,21 +37,18 @@ private Example()
...
             Log.Warn($"I correctly read the string config, its value is: {Config.String}");
             Log.Warn($"I correctly read the int config, its value is: {Config.Int}");
             Log.Warn($"I correctly read the float config, its value is: {Config.Float}");
+            base.OnEnabled();
```
Comment (iRebbok) [inline Exiled.Example/Example.cs:44]: Add a line breaker above that line.

**Review by iRebbok — CHANGES_REQUESTED** [review]
The private constructor used here to tell to Exiled to use one from a static property instead. Revert deletion.

**Review by iRebbok — APPROVED** [review]

**Comment by iRebbok** [comment]
Ignore CI, it fails due to a certificate issue on www.exiled.host.

**Review by iopietro — APPROVED** [review]

**Pull request merged**